code/__DEFINES/atmospherics/atmos_helpers.dm

```diff
@@ -3,6 +3,13 @@
 ///(V) is if the share is vertical or not. True or False
 #define CANATMOSPASS(A, O, V) ( A.can_atmos_pass == ATMOS_PASS_PROC ? A.can_atmos_pass(O, V) : ( A.can_atmos_pass == ATMOS_PASS_DENSITY ? !A.density : A.can_atmos_pass ) )
 
+/// Get the amount of force this atom (A) can resist from turf (T)
+/// Allows for proc overrides if the atom does something unique
+#define GETPRESSURERESIST(A, T) ( A.complex_pressure ? A.get_pressure_resistance(T) : A.pressure_resistance )
```

Comment:
SCREAMING_SNAKE_CASE()

Reply:
My ancestors are smiling on my imperial, can you say the same?